### PR TITLE
[BillingRP] New API to list transactions by invoice

### PR DESCRIPTION
**Pull request by asarkar84**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 1]
```diff
@@ -2723,6 +2723,51 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/invoices/{invoiceName}/transactions": {
+      "get": {
```
Comment (pilor) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: >"get" [](start = 6, length = 5)

Should contain the x-ms-pageable flag to let clients know how to follow nextLinks.  [https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-pageable)

**Review by pilor — CHANGES_REQUESTED** [review]
:clock1:

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json (hunk 1 above)
Comment (asarkar84) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: Sure! done.

**Review by pilor — APPROVED** [review]

**Comment by asarkar84** [comment]
@NullMDR Please merge the PR.

**Comment by NullMDR** [comment]
@asarkar84 Please fix the spellcheck

**Pull request merged**